lib/tapioca/dsl/compilers/job_iteration.rb

```diff
@@ -0,0 +1,96 @@
+# typed: strict
+# frozen_string_literal: true
+
+return unless defined?(JobIteration::Iteration)
+
+require "prism"
```

Comment:
We should not need Prism in DSL compilers at all. Why do we need this?

Reply:
Updated.